inception-workload/src/main/java/de/tudarmstadt/ukp/inception/workload/settings/WorkloadPanel.java

```diff
@@ -27,63 +27,55 @@
 import org.apache.wicket.markup.html.panel.Panel;
 import org.apache.wicket.model.IModel;
 import org.apache.wicket.model.Model;
-import org.apache.wicket.model.StringResourceModel;
 import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import de.agilecoders.wicket.core.markup.html.bootstrap.form.BootstrapRadioChoice;
 import de.tudarmstadt.ukp.clarin.webanno.model.Project;
 import de.tudarmstadt.ukp.clarin.webanno.support.lambda.LambdaAjaxButton;
-import de.tudarmstadt.ukp.inception.workload.dynamic.manager.WorkflowProperties;
-import de.tudarmstadt.ukp.inception.workload.dynamic.manager.WorkloadProperties;
+import de.tudarmstadt.ukp.inception.workload.model.WorkloadManagementService;
+import de.tudarmstadt.ukp.inception.workload.registry.WorkloadRegistry;
 
 //Custom panel inside the page
-public class WorkflowAndMonitoringPanel extends Panel
+public class WorkloadPanel extends Panel
 {
     private static final long serialVersionUID = -6220828178550562376L;
 
-    //SpringBeans
-    private @SpringBean WorkloadProperties workloadProperties;
-    private @SpringBean WorkflowProperties workflowProperties;
-
-    private final List<String> workflow;
-    private final BootstrapRadioChoice<String> workflowChoices;
     private final List<String> monitoring;
     private final BootstrapRadioChoice<String> monitoringChoices;
+    private final Project project;
+
+    private @SpringBean WorkloadManagementService workloadManagementService;
+    private @SpringBean WorkloadRegistry workloadRegistry;
 
-    public WorkflowAndMonitoringPanel(String aID, IModel<Project> aProject) {
+    @Autowired
+    public WorkloadPanel(String aID, IModel<Project> aProject)
+    {
         super(aID, aProject);
 
+        project = aProject.getObject();
+
         //Basic form
         Form<Void> form = new Form<>("form");
 
-        //Add two possibilities to select for the workflow manager
-        workflow = new ArrayList<>();
-        workflow.add("Default workflow manager");
-        workflow.add("Dynamic workflow manager");
-
-        workflowChoices = new BootstrapRadioChoice<>("workflowRadios",
-            new Model<>(getString("defaultWorkflow")), workflow);
-        workflowChoices.setInline(true);
-        workflowChoices.setOutputMarkupId(true);
-        //Set default for the group
-        workflowChoices.setModel(new Model<>(workflowProperties.getProperty()));
-
-        //add them to the form
-        form.add(workflowChoices);
-
         //Add two possibilities to select from the monitoring manager
         monitoring = new ArrayList<>();
```

Comment:
The list of available managers must be obtained from the registry. It cannot be hard-coded here.